src/component/Filter.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { SelectedCategory } from '../types/SelectedCategory';
+
+type Props = {
+  category: SelectedCategory;
+  onClick: (selectedCategory: SelectedCategory) => void,
+};
+
+export const Filter: React.FC<Props> = ({ category, onClick }) => {
+  return (
+    <nav className="filter">
+      <a
+        href="#/"
+        className={cn('filter__link', {
+          selected: category === SelectedCategory.All,
+        })}
+        onClick={() => onClick(SelectedCategory.All)}
+      >
+        {SelectedCategory.All}
+      </a>
+
+      <a
+        href="#/active"
+        className={cn('filter__link', {
+          selected: category === SelectedCategory.Active,
+        })}
+        onClick={() => onClick(SelectedCategory.Active)}
+      >
+        {SelectedCategory.Active}
+      </a>
+
+      <a
+        href="#/completed"
+        className={cn('filter__link', {
+          selected: category === SelectedCategory.Completed,
+        })}
+        onClick={() => onClick(SelectedCategory.Completed)}
+      >
+        {SelectedCategory.Completed}
+      </a>
+    </nav>
+  );
```

Comment:
Consider to create handler
```suggestion
export const Filter: React.FC<Props> = ({ category, onClick }) => {
 
  const onCategorySelect = (category: SelectedCategory) => () => {
    onClick(category);
  };
  
  return (
    <nav className="filter">
      <a
        href="#/"
        className={cn('filter__link', {
          selected: category === SelectedCategory.All,
        })}
        onClick={onCategorySelect(SelectedCategory.All)}
      >
        {SelectedCategory.All}
      </a>

      <a
        href="#/active"
        className={cn('filter__link', {
          selected: category === SelectedCategory.Active,
        })}
        onClick={onCategorySelect(SelectedCategory.Active)}
      >
        {SelectedCategory.Active}
      </a>

      <a
        href="#/completed"
        className={cn('filter__link', {
          selected: category === SelectedCategory.Completed,
        })}
        onClick={onCategorySelect(SelectedCategory.Completed)}
      >
        {SelectedCategory.Completed}
      </a>
    </nav>
  );
```